src/main/java/edu/kit/ipd/crowdcontrol/objectservice/database/operations/AnswerRatingOperations.java

```diff
@@ -45,13 +56,31 @@ public AnswerRecord insertNewAnswer(AnswerRecord answerRecord) {
      */
     public RatingRecord insertNewRating(RatingRecord ratingRecord) {
         ratingRecord.setIdRating(null);
-        return doIfRunning(ratingRecord.getExperiment(), conf ->
+        RatingRecord result = doIfRunning(ratingRecord.getExperiment(), conf ->
                 DSL.using(conf)
                         .insertInto(RATING)
                         .set(ratingRecord)
                         .returning()
                         .fetchOne()
         );
+        addToExperimentCalibration(ratingRecord.getWorkerId(), ratingRecord.getExperiment());
+        return result;
+    }
+
+    private void addToExperimentCalibration(int workerID, int experimentId) {
```

Comment:
yes...and no ;) Every experiment has a calibration associated with it, where every worker working on an experiment can gets associated with (answered the the only question of the calibration). You can then exclude the workers which have previously worked on an experiment via the same mechanism as you would exclude a worker with the wrong calibration. (the automatic calibration gets excluded in the normal get calibrations-call). It's reducing the complexity in the database, os and the workerservice. Maybe i should write some javadoc!